sql/core/src/main/scala/org/apache/spark/sql/api/python/PythonSQLUtils.scala

```diff
@@ -44,6 +48,13 @@ private[sql] object PythonSQLUtils {
     val conf = new SQLConf()
     // Force to build static SQL configurations
     StaticSQLConf
+    try {
+      val symbol = ScalaReflection.mirror.staticModule("org.apache.spark.sql.hive.HiveUtils")
```

Comment:
Could you leave some comments here like `// Force to build Hive configuration`?

Reply:
`// Force to build SQL configurations from Hive module` may be better since they are not hive configs

Reply:
yea, ok.